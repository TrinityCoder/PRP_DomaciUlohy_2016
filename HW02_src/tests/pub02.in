140 128 8 1603
4905 -3 -2 0 0
9166 255 -1322
8900 10000 -1 6543
