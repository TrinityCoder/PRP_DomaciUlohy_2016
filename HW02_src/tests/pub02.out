140, 128, 8, 1603, 4905, -3, -2, 0, 0, 9166, 255, -1322, 8900, 10000, -1, 6543
Pocet cisel: 16
Pocet kladnych: 10
Pocet zapornych: 4
Procento kladnych: 62.50
Procento zapornych: 25.00
Pocet sudych: 10
Pocet lichych: 6
Procento sudych: 62.50
Procento lichych: 37.50
Prumer: 2520.00
Maximum: 10000
Minimum: -1322
